packages/synapse-react-client/src/components/RecentPublicationsGrid/RecentPublicationsGrid.tsx

```diff
@@ -0,0 +1,176 @@
+      >
+        {dataRows.map(pub => (
+          <Grid item xs={12} sm={6} md={4} key={pub.rowId}>
+            <Card sx={{ height: '100%', boxShadow: 'none', border: 'none' }}>
+              <CardContent>
+                <Box display={'flex'} gap={'6px'} flexWrap={'wrap'}>
+                  {pub.values[tagColIndex] &&
+                    JSON.parse(pub.values[tagColIndex] as string).map(
+                      (tag: string, index: number) => (
+                        <Typography
+                          key={index}
+                          variant="overline"
+                          sx={{
+                            backgroundColor: '#EAECEE',
+                            borderRadius: '3px',
+                            padding: '4px 8px',
+                            border: 'none',
+                            lineHeight: 'initial',
+                          }}
+                        >
+                          {tag}
+                        </Typography>
+                      ),
+                    )}
+                </Box>
+                <Typography
+                  variant="headline2"
+                  sx={{
+                    padding: '20px 0px',
+                  }}
+                >
+                  {pub.values[titleColIndex]}
+                </Typography>
+                <Typography
+                  variant="body1"
+                  color="#71767F"
+                  sx={{
+                    fontStyle: 'italic',
+                    fontSize: '14px',
+                  }}
+                >
+                  {pub.values[journalColIndex]}
+                </Typography>
+                <Typography
+                  variant="body1"
+                  color="#71767F"
+                  sx={{ fontSize: '14px' }}
+                >
+                  {pub.values[publicationDateColIndex] &&
+                    formatDate(
+                      dayjs(Number(pub.values[publicationDateColIndex])),
+                      'MMMM, YYYY',
+                    )}
+                </Typography>
+              </CardContent>
+            </Card>
+          </Grid>
+        ))}
+      </Grid>
+
+      <Box
+        display="flex"
+        flexDirection="column"
+        gap="16px"
+        sx={{
+          height: '4px',
+          borderTop: '3px solid #DFE2E6',
+          alignItems: 'flex-start',
+          flex: isSmallScreen || isMediumScreen ? '0 0 100%' : '1',
+          padding: isSmallScreen || isMediumScreen ? '0 0 0 16px' : '0',
+        }}
+      >
+        <Typography variant="headline2" paddingTop="26px">
+          Recently Published
+        </Typography>
+        <Typography
+          variant="body1"
+          sx={{ fontStyle: 'italic', color: '#4A5056' }}
+        >
+          {summaryText && summaryText}
+        </Typography>
+        {buttonLink && buttonLinkText && (
+          <Button
+            variant="contained"
+            color="primary"
+            href={buttonLink}
```

Comment:
this button will keep you in the portal, so can you add a prop `component={Link}` where `Link` is from react-router-dom, and change the `href` prop to `to`? This would let us use react-router navigation, which would prevent a full page reload when this is clicked.